<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="VaraLabel">
        <property name="text">
         <string>Variable a</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="VarbLineEdit">
        <property name="text">
         <string>0</string>
        </property>
        <property name="placeholderText">
         <string>Enter variable b</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="TOFLabel">
        <property name="text">
         <string>Type of a function</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="TOFComboBox">
        <item>
         <property name="text">
          <string>Linear (a*x+b)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Radical (a/x+b)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Modulus (a*|x|+b)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Power (a*x^c+b)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sinus (a*sin(x)+b)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cosinus (a*cos(x)+b)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tangent (a*tg(x)+b)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="VaraLineEdit">
        <property name="text">
         <string>1</string>
        </property>
        <property name="placeholderText">
         <string>Enter variable a</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="VarbLabel">
        <property name="text">
         <string>Variable b</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="DrawButton">
        <property name="text">
         <string>Draw</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCustomPlot" name="Graph" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
